yarn.lock

```diff
@@ -4568,7 +4568,7 @@ hosted-git-info@^6.0.0:
   dependencies:
     lru-cache "^7.5.1"
 
-html-entities@^2.3.2:
+html-entities@^2.3.2, html-entities@^2.4.0:
```

Comment:
Can you update us to using 2.4.0 as well? Might as well not double up deps

Reply:
Happily! That appears to be coming from eslint-plugin-libram; is the easiest fix then to change eslint-plugin-libram's dependencies?